conf/messages.xml

```diff
@@ -0,0 +1,3221 @@
+<?xml version="1.0"?>
+<!DOCTYPE protocol SYSTEM "messages.dtd">
```

Comment:
remove this file from conf folder
if you need a new message, make a pull request in pprzlink and update the reference in paparazzi submodule